ui/start.html

```diff
@@ -1327,6 +1327,22 @@ <h1 class="js-i18n js-terms-title"></h1>
         </section>
       </section>
 
+      <section class="viewport are_you_sure_state" data-tr-section="are-you-sure">
+        <section class="info">
+          <h1 class="js-i18n">Are You Sure?</h1>
```

Comment:
I'm not sure how i18n is actually working on this project. Should I add the texts on this new screen to some file?

Reply:
No need to add texts to a file. The ``js-i18n`` class marks this as a screen to be translated.